.asciidoctor/diagram/diagram-classes.png.cache

```diff
@@ -0,0 +1 @@
+{"checksum":"57260c74db70c76d3c58df3c219b543e","options":{"size_limit":"4096"},"width":812,"height":525}
```

Comment:
This file should be removed since it's gitignored.

Reply:
fixed